app/src/main/java/com/bunbeauty/papakarlo/feature/auth/LoginFragment.kt

```diff
@@ -49,7 +49,7 @@ class LoginFragment : BaseComposeFragment<Login.State, Login.Action, Login.Event
 
     private val args: LoginFragmentArgs by navArgs()
 
-    private val LogoBoxHeightLimit: Dp = 200.dp
+    private val logoBoxHeightLimit: Dp = 200.dp
```

Comment:
Let's change it back according Compose [code style](https://github.com/androidx/androidx/blob/androidx-main/compose/docs/compose-api-guidelines.md#singletons-constants-sealed-class-and-enum-class-values)